components/TodoList.tsx

```diff
@@ -1,3 +1,35 @@
-export function TodoList() {
-  return <></>;
+import { Text, ListItem, Stack } from "tamagui";
+
+import { TodoContext, TodoStore } from "../stores/Todo.store";
+import { observer } from "mobx-react-lite";
+import { useContext } from "react";
+
+function EmptyTodo() {
```

Comment:
Good practice is to have one component per file.

Reply:
I know that, but these seem very small and should not be exported to the rest of the tree. Creating a file for each would  also lead to duplicating the imports and having to switch between files to work on them at all.